tests/unit/legacy-0-6-x/test-module-for-component-test.js

```diff
@@ -475,8 +475,9 @@ test('it supports updating an input', function(assert) {
   let input = this._element.querySelector('input');
   input.value = '1';
 
-  fireEvent(input, 'change');
-  assert.equal(this.get('value'), '1');
+  return triggerEvent(input, 'change').then(() => {
+    assert.equal(this.get('value'), '1');
+  });
```

Comment:
seems like we should `assert.expect(1)` here to make sure it actually runs correctly